tests/framework/extensions/clusters/kubernetesversions/default.go

```diff
@@ -0,0 +1,72 @@
+package kubernetesversions
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	"github.com/rancher/rancher/tests/framework/extensions/clusters"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+)
+
+func Default(t *testing.T, client *rancher.Client, provider string, kubernetesVersions []string) ([]string, error) {
```

Comment:
I guess it's too late now, but this testing t parameter is unnecessary

Reply:
Could you make a quick update Josh? And make sure the backport reflects the same.

Reply:
yes 👍